src/components/get-games.tsx

```diff
@@ -0,0 +1,47 @@
+// EXAMPLE
+
+import { getGames } from "@/lib/firebase_utils";
```

Comment:
Is this import now correct? When you changed the filename to `firebase.utils.ts`?